### PR TITLE
Trivy image scan: Vulnerability location

### PR DESCRIPTION
**Pull request by Yuan-Yubin**
The image scanning results don't show the location of vulnerable components.
The trivy scan report contains the component path information, "PkgPath" is the file path.
The structure of finding, including "file_path" field, which can also correspond to the path of the image scanning vulnerability file.

### CODE REVIEW

File: dojo/tools/trivy/parser.py [hunk 1]
```diff
@@ -126,6 +126,7 @@ def get_result_items(self, test, results, service_name=None):
                     vuln_id = vuln.get('VulnerabilityID', '0')
                     package_name = vuln['PkgName']
                     severity = TRIVY_SEVERITIES[vuln['Severity']]
+                    file_path = vuln['PkgPath']
```
Comment (damiencarol) [inline dojo/tools/trivy/parser.py:129]:
```suggestion
                    file_path = vuln.get('PkgPath')
```

**Review by damiencarol — CHANGES_REQUESTED** [review]
Please add a unit test to test these new data

File: dojo/tools/trivy/parser.py (hunk 1 above)
Comment (Yuan-Yubin) [reply dojo/tools/trivy/parser.py:129]: Do I need to write a new test case? Or use the given tests?

Comment (damiencarol) [reply dojo/tools/trivy/parser.py:129]: both options are good, just add new checks somewhere that test the new data

**Comment by jerrinss5** [comment]
@damiencarol and/or @Yuan-Yubin i have added test cases for this one to my fork. How do i contribute on to this PR? https://github.com/jerrinss5/django-DefectDojo/commit/3de2e6152d4f5f32e630185ff9d3f3c9ee5d8466

**Comment by jerrinss5** [comment]
@damiencarol and/or @Yuan-Yubin i have added test cases for this one to my fork. How do i contribute on to this PR? https://github.com/jerrinss5/django-DefectDojo/commit/3de2e6152d4f5f32e630185ff9d3f3c9ee5d8466

**Comment by Maffooch** [comment]
@jerrinss5 you would need to commit your changes to Yuan-Yubin:yuanyubin

**Comment by jerrinss5** [comment]
thanks @Maffooch for your reply. I wasn't able to figure out how to commit changes to Yuan-Yubin:yuanyubin. So created a new pull request instead with both the changes combined. Hope that is ok. https://github.com/DefectDojo/django-DefectDojo/pull/7076

**Comment by Maffooch** [comment]
Thanks for the #7076 @jerrinss5!

**Pull request closed without merging**